app/views/products/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_for(product) do |f| %>
+  <% if product.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(product.errors.count, "error") %> prohibited this product from being saved:</h2>
+
+      <ul>
+      <% product.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :description %>
+    <%= f.text_field :description %>
+  </div>
+
+  <div class="field">
+    <%= f.label :picture %>
+    <%= f.text_field :picture %>
+  </div>
+
+  <div class="field">
+    <%= f.label :unit_price %>
+    <%= f.number_field :unit_price, class: :text_field, step: :any%>
```

Comment:
I'm concerned about leaving the step as any. I assume that it will default to 1 (integers). Even if it still accepts floats, I think we need to look carefully at this implementation in terms of if we want the arrows to make it go up by cents or dollars... because it's probably going to go up by dollars right now.